src/mlpack/bindings/python/tests/test_python_binding_main.cpp

```diff
@@ -105,6 +109,24 @@ static void mlpackMain()
     CLI::GetParam<arma::Mat<size_t>>("umatrix_out") = move(out);
   }
 
+  // An input matrix(pandas.Series) should have all elements multiplied by two.
+  if (CLI::HasParam("smatrix_in"))
+  {
+    arma::mat out = move(CLI::GetParam<arma::mat>("smatrix_in"));
+    out *= 2.0;
+
+    CLI::GetParam<arma::mat>("smatrix_out") = move(out);
+  }
+  // An input matrix(pandas.Series) should have all elements multiplied by two.
+  if (CLI::HasParam("s_umatrix_in"))
+  {
+    arma::Mat<size_t> out =
+        move(CLI::GetParam<arma::Mat<size_t>>("s_umatrix_in"));
+    out.row(0)*= 2.0;
```

Comment:
You should be able to just do `out *= 2` here, assuming your intention is to multiply every element by 2 (I think that all inputs to `smatrix_in` and `s_umatrix_in` have only one row).